sample_apps/run_example.sh

```diff
@@ -515,6 +524,13 @@ function is_valid_step() {
         return
     fi
 
+    # Special-hook to run simple_app to exercise diff crypto algorithms
+    if [ "${SampleAppName}" == "simple_app" ] \
+    && [ "${fn}" == "run_test-crypto_algorithms" ]; then
```

Comment:
We now will allow this as a valid invocation: `$ ./run_example.sh simple_app run_test-crypto_algorithms`